scripts/cnode-helper-scripts/env

```diff
@@ -250,7 +258,7 @@ BLOCKLOG_DB="${BLOCKLOG_DIR}/blocklog.db"
 [[ -z "${IP_VERSION}" ]] && IP_VERSION=4
 IP_VERSION=$(tr '[:upper:]' '[:lower:]' <<< "${IP_VERSION}")
 [[ -z ${USE_EKG} ]] && USE_EKG='Y'
-CNODE_PID=$(pgrep -fn "[c]ardano-node.*.--port ${CNODE_PORT}")
+CNODE_PID=$(pgrep -fn "$(basename ${CNODEBIN}).*.--port ${CNODE_PORT}")
```

Comment:
In all other places ${CNODEBIN} is used directly, any reason to do otherwise here?

Reply:
Hmm I actually wonder, should we use basename at all places instead?
Reason being that will allow to not assume that the execution command contains full path for cardano-node.

Reply:
makes sense 👍